ts/main.ts

```diff
@@ -48,17 +52,44 @@ $entryForm.addEventListener('submit', (event: Event) => {
   const title = $titleInput.value;
   const photoUrl = $photoUrlInput.value;
   const notes = $notesTextArea.value;
-  const entryValues: Entry = {
-    title,
-    photoUrl,
-    notes,
-    entryId: data.nextEntryId,
-  };
-  data.nextEntryId++;
-  data.entries.unshift(entryValues);
-  writeData();
-  const $newEntry = renderEntry(entryValues);
-  $allEntriesUl.prepend($newEntry);
+  if (data.editing === null) {
+    const entryValues: Entry = {
+      title,
+      photoUrl,
+      notes,
+      entryId: data.nextEntryId,
+    };
+    data.nextEntryId++;
+    data.entries.unshift(entryValues);
+    writeData();
+    const $newEntry = renderEntry(entryValues);
+    $allEntriesUl.prepend($newEntry);
+  } else {
+    const entryValues: Entry = {
+      title,
+      photoUrl,
+      notes,
+      entryId: data.editing.entryId,
+    };
```

Comment:
Instead of making the variables inside your condition (line 56 and line 68), why not move this above your condition and just change the value of the property if we are editing? It will cut down on lines a little bit and be a little cleaner since all we need to do is modify the `entryId` property.